src/main/java/hunternif/mc/impl/atlas/marker/GlobalMarkersDataHandler.java

```diff
@@ -56,7 +56,7 @@ public GlobalMarkersData getData() {
 
 	/** Synchronizes global markers with the connecting client. */
 	public void onPlayerLogin(ServerPlayerEntity player) {
-		data.syncOnPlayer(player);
+		if(data != null) data.syncOnPlayer(player);
```

Comment:
This is a workaround for #309, which is the same issue as BIOX in discord has, right?

I'm wondering why `data` is `null`. Must have something to do with a different event order in forge?

Reply:
Maybe it's a dimension thing. after all `data` is only set during world load when the world loaded is the overworld, so at that point, `data` would be null if for some reason the overworld was not loaded. Though it's just a hypothesis as I've never really used mods that mess with the game that way.